### ISSUE TITLE
Cannot find `x-goog-api-key` header for Google auth

### ISSUE THREAD

**Issue by akivajgordon** [chrisjshull/homebridge-nest#146]
The README [instructions for Google authentication](https://github.com/chrisjshull/homebridge-nest#using-a-google-account) are great, but after following the instructions for getting the Google API key, I can't find the `x-goog-api-key` header. The instructions read:

> 11. In the 'Filter' box, enter issue_jwt
> 12. Click on the last issue_jwt call.
> 13. In the Headers tab, under Request Headers, copy the entire x-goog-api-key (do not include the x-goog-api-key: name). This is your "apiKey" in config.json.

I do see the "issue_jwt" call, but __I do not see the `x-goog-api-key` header anywhere__. I tried multiple times from both Safari and Chrome, and that header is never there. Is there another way for me to retrieve the API key? This is a blocker for me with respect to homebridge-nest, because I cannot otherwise authenticate. I've already migrated my Nest account with Google, so I can't connect with my Nest credentials anymore.

In short, how can I retrieve the Google API key for the config?

**Comment by adriancable** [comment]
@akivajgordon - please send me an email at adrian.cable@gmail.com and I'll get you going. Thanks.

**Comment by Thomas034** [comment]
Hi Adrian, i have a problem with config, this is error reported to starting homebridge -D
![Annotazione 2019-08-26 163047](https://user-images.githubusercontent.com/54308077/63698380-01b9b580-c81f-11e9-8abf-646a5a113087.png)
 and this is my config on config.json
{
            "platform": "Nest",
            "googleAuth": {
                "issueToken": "https://accounts.google.com/o/oauth2/.........",
                "cookies": "OCAK=WEq8...............",
                "apiKey": "AIza..............."
            }
        },
where is the problem?
Thanks in advice for your reply and sorry for my bad english...

**Comment by legmar** [comment]
This problem (missing x-goog-api-key in the request headers when filtering by issue_jwt) also occurred for me.  The explanation for this behavior is that you have not finished migration and setup of the linking from your Nest account to your Google Account.  After the login pages, you must continue to click next and complete all the migration steps.  Once all the steps were completed, a 5th issue_jwt showed up in the list with a request header that contained x-google-api-key.  Hope this helps someone!

**Comment by brando-soto28** [comment]
I am having this same issue. Any assistance? What do you need from me?

**Comment by ssllii** [comment]
I also cannot find the x-goog-api-key in the request headers.  I can see only 2 issue_jwt.
Also the "home" does not match the content of the Google Home app, so I am not sure whether there is a "migration" issue or not.
